tests/baselines/reference/docker/azure-sdk.log

```diff
@@ -147,6 +144,15 @@ npm ERR! Failed at the @azure/keyvault-certificates@X.X.X extract-api script.
 npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
 npm ERR! A complete log of this run can be found in:
 npm ERR!     /root/.npm/_logs/XXXX-XX-XXXXXXXXX-debug.log
+@azure/service-bus (? seconds)
+>>> @azure/service-bus
+tsc -p . && rollup -c 2>&1 && npm run extract-api
+src/serviceBusClient.ts(90,34): error TS1109: Expression expected.
+src/serviceBusClient.ts(90,52): error TS1109: Expression expected.
+src/serviceBusClient.ts(90,62): error TS1005: ':' expected.
+src/serviceBusClient.ts(92,52): error TS1109: Expression expected.
+src/serviceBusClient.ts(92,70): error TS1109: Expression expected.
+src/serviceBusClient.ts(92,98): error TS1005: ':' expected.
```

Comment:
this comes from `?.` and indicates that we're compiling with 3.6 or earlier.

Reply:
Perhaps https://github.com/microsoft/TypeScript/blob/38e1e660c8ca1f4e75100d38636666569d105da0/tests/cases/docker/azure-sdk/Dockerfile#L9 ?